src/jobs/build.yml

```diff
@@ -35,6 +35,18 @@ parameters:
     description: "Additional flags to pass when running Hugo (e.g. -DF)."
     type: string
     default: ""
+  htmlproofer-http-status-ignore:
+    description: "An array of numbers representing status codes to ignore."
+    type: string
+    default: "''"
+  htmlproofer-url-ignore:
+    description: "An array of Strings or RegExps containing URLs that are safe to ignore. It affects all HTML attributes. Note that non-HTTP(S) URIs are always ignored."
+    type: string
+    default: "''"
+  htmlproofer-timeframe:
+    description: "A string representing the caching timeframe. The format is `[0-9]+[mhd]` where `m` stands for month, `h` for hours and `d` for days."
+    type: string
+    default: "'7d'"
```

Comment:
This should match the same as the command's default of `"'1hr'"`

Reply:
Updated in commit 344c2cd11f763d2160c3eb4df4c54a42d38a6688.